main.py

```diff
@@ -0,0 +1,15 @@
+# this allows us to use code from
+# the open-source pygame library
+# throughout this file
+import pygame
+
+
+def main():
+    try:
+        pygame.init()
+        print('Starting asteroids!')
+    except Exception as e:
+        print(f'Failed to initialize Pygame: {e}')
+
+if __name__ == "__main__":
+    main()
```

Comment:
**Suggestion:** Add a newline at the end of the file to comply with PEP 8.

**Existing Code:**
```Python
main()
```

**Improved Code:**
```Python
main()


```

**Details:**
Add a newline at the end of the file to adhere to PEP 8 style guidelines.